generator.yml

```diff
@@ -1002,7 +996,7 @@ builds:
   - {<< : *ppc64le,           << : *stable-5_10,      << : *llvm,            boot: true,  llvm_version: *llvm_11}
   # https://github.com/ClangBuiltLinux/linux/issues/1023
   # https://github.com/ClangBuiltLinux/linux/issues/1143
```

Comment:
same

Reply:
This one is fine because I did not touch 5.10.